tutorials/01_intro.md

```diff
@@ -2,18 +2,61 @@
-goal of the library is to provide an abstraction layer to various physics
-engines, which gives end users the ability to use multiple physics engines
-with minimal change to their application code. Ignition Physics uses
-a plugin architecture where each physics engine is implemented as a plugin
-that can be loaded at runtime. To enable users the ability to choose
-between physics engines, Ignition Physics introduces [\"Features\"](@ref ignition::physics::Feature), a design concept used to encode the
-capabilities of a physics engine. Users can request for physics engines
-that support a set of features and the plugin loading mechanism loads only
-the engines that implement the requested features.
-
-The following is a list of currently supported physics engines:
-
-  * [DART](http://dartsim.github.io/)
+## Motivation
+
+Ignition Physics (`ign-physics`) is a library component in Ignition, a set of libraries designed to rapidly develop robot and simulation applications.
+The main goal of the library is to provide an abstraction layer to various physics engines, which gives end-users the ability to use multiple physics engines with minimal change to their application code.
+Users can request for physics engines that support a set of features and the plugin loading mechanism loads only the engines that implement the requested features.
+Besides, a user-selected physics engine can be integrated into the existing code-base by writing a compatible plugin interface, which enables Ignition Physics extensibility and modularity.  
+
+## Overview
+
+For a big picture of the Ignition Physics operation in Ignition ecosystem, see the abstract diagram below:
+
+<img src="https://user-images.githubusercontent.com/18066876/94050480-9dc54000-fdd6-11ea-92e7-832dcdf4caba.png"/>
+
+In general, `ign-gazebo` is the main simulation library, in which its functionalities are powered by many component libraries.
+For example, its graphical drawing is supported by `ign-rendering` or simulated sensors that are defined and implemented in `ign-sensors`.
+In particular, this library `ign-physics` provides an abstract interface to physics engines, which simulates dynamic transformations and interactions of objects in `ign-gazebo`.
+The communication between these libraries at runtime is provided by `ign-transport` and `ign-msgs`.
+
+Ignition Physics uses a plugin architecture where each physics engine is implemented as a plugin that can be loaded at runtime.
+To enable users the ability to choose between physics engines, Ignition Physics introduces four keys elements:
+
+- \ref ignition::physics::Entity "Entity": the base class, which abstracts all proxy objects, which contains minimal descriptive data (e.g. a unique identifier, a reference-counter, etc.) pointed the corresponding implemented objects.
+- \ref ignition::physics::FeaturePolicy "FeaturePolicy": the policy class, which provides metadata to features about their simulation engine specifications.
```

Comment:
I would mention the two types of metadata currently supported by the `FeaturePolicy` class:

* Scalar numeric type (64-bit `double` or 32-bit `float`)
* Dimensionality (2D or 3D)

I've seen numerous physics engines that support different scalar types, but I've never seen one that supports both 2D and 3D